src/processing-manager.ts

```diff
@@ -21,7 +21,8 @@ export class ProcessingManager {
         blobAccount: "wetstorage",
         blobContainer: "websites",
         processes: os.cpus().length,
-        crawlVersion: "CC-MAIN-2017-13"
+        crawlVersion: "CC-MAIN-2017-13",
```

Comment:
I don't understand your question. This is just to specify which common crawl data set should be used. The identifiers for all the data sets are available on the CC website.